ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -317,9 +326,48 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
       history.back();
     }
 
+    async initialBillingAccountLoad() {
+      // If user hasn't grant GCP billing scope to workbench, we can not fetch billing account from Google
+      // or fetch user's available billing accounts.
+      // When creating/duplicating workspace, show free tier billing account.
+      // When editing existing workspace, show free tier if that is currently being used or 'User Provided Billing Account'
+      // if it is user's billing account.
+      if (serverConfigStore.get().config.enableBillingUpgrade && !hasBillingScope()) {
+        const freeTierBillingAccount: BillingAccount = {
+          name: 'billingAccounts/' + serverConfigStore.get().config.freeTierBillingAccountId,
+          isFreeTier: true,
+          isOpen: true,
+          displayName: 'Use All of Us initial credits',
+        };
+        if (this.isMode(WorkspaceEditMode.Create) || this.isMode(WorkspaceEditMode.Duplicate)) {
+          this.setState(prevState => fp.set(
+              ['workspace', 'billingAccountName'],
+            freeTierBillingAccount.name,
+            prevState));
+          this.setState({'billingAccounts': [freeTierBillingAccount]});
```

Comment:
nit: in general, would prefer normal keys over string keys - here and throughout. I would only use string keys if I'm using an object as generic key/value lookup